integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/ElasticsearchExtensionIT.java

```diff
@@ -310,12 +316,12 @@ public void query_explain_multipleIndexes_invalidIndexName() {
 				.toQuery();
 
 		Assertions.assertThatThrownBy(
-				() -> query.explain( "NotAnIndexName", FIRST_ID )
+				() -> query.explain( "NotAMappedName", FIRST_ID )
 		)
 				.isInstanceOf( SearchException.class )
 				.hasMessageContaining(
-						"type name 'NotAnIndexName' is not among the mapped type targeted by this query: ["
-						+ INDEX_NAME + ", " + OTHER_INDEX_NAME + "]"
+						"type name 'NotAMappedName' is not among the mapped type targeted by this query: [",
+						INDEX_NAME, OTHER_INDEX_NAME
```

Comment:
Shouldn't this be `TYPE_NAME`, `OTHER_TYPE_NAME`?

Also, why did you have to remove the comma and square bracket? Something looks wrong here.

Reply:
Yes they should. Same issue as above :)